SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -275,6 +276,39 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 	}
 }
 
+void USpatialNetDriver::CreateServerWorkerEntity()
+{
+	const FString WorkerId = FString(TEXT("workerId:")) + Connection->GetWorkerId();
```

Comment:
maybe this should be `workerEntity:`, then when we document this is in the glossary / other docs it's a more logically connection

Reply:
The permission needs to stay like this. I assume you propose changing the name of the entity displayed in the inspector which comes from the `Metadata` component. I'm happy to change this.